nipype/pipeline/engine/tests/test_utils.py

```diff
@@ -18,6 +18,11 @@
 from ..utils import merge_dict, clean_working_directory, write_workflow_prov
 
 
+def test_traits_version():
+    #just a temprorary test, testing CI
+    import traits
+    assert traits.__version__ >= "4.6.0"
```

Comment:
Agreed. Actually, we should have like a pre-install hook (we've talked about it) to check versions.